FakeNFT/Scenes /ProfileEdit/ProfileEditPresenterProtocol.swift

```diff
@@ -0,0 +1,66 @@
+//
+//  ProfileEditPresenterProtocol.swift
+//  FakeNFT
+//
+//  Created by Alibi Mailan on 01.03.2025.
+//
+
+import UIKit
+import ProgressHUD
+
+protocol ProfileEditPresenterProtocol: AnyObject {
+    func viewDidLoad()
+    func didTapSave(name: String, description: String, website: String, photo: UIImage?)
+    func didTapChangePhoto()
+    func didTapClose()
+    func profileLoaded(_ profile: ProfileEntity)
+    func profileLoadFailed(error: Error)
+    func profileSaveSucceeded()
+    func profileSaveFailed(error: Error)
+}
+
+final class ProfileEditPresenter: ProfileEditPresenterProtocol {
+    weak var view: ProfileEditViewProtocol?
+    var interactor: ProfileEditInteractorInput!
+    var router: ProfileEditRouterProtocol!
+    
+    func viewDidLoad() {
+        view?.showLoadingIndicator()
+        interactor.loadProfile()
+    }
+    
+    func didTapSave(name: String, description: String, website: String, photo: UIImage?) {
```

Comment:
Методы с большим количеством аргументов не очень удобно читать, когда они в одну строчку написаны, мне на ревью так рекомендовали делать:

```suggestion
    func didTapSave(
        name: String, 
        description: String, 
        website: String, 
        photo: UIImage?
    ) {
```